### PR TITLE
Change peerDependencies to dependencies #584

### PR DESCRIPTION
**Pull request by ruarfff**
## Hey, I just made a Pull Request!

Change peerDependencies to dependencies and removed some duplication.

Addresses #584

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [x] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: plugins/home-page/package.json [hunk 1]
```diff
@@ -8,27 +8,21 @@
...
-    "@types/testing-library__jest-dom": "5.0.2",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0"
+    "@types/testing-library__jest-dom": "5.0.2"
   },
-  "peerDependencies": {
-    "@backstage/core": "^0.1.1-alpha.3",
-    "@backstage/theme": "^0.1.1-alpha.3",
+  "dependencies": {
+    "@backstage/core": "^0.1.1-alpha.4",
```
Comment (Rugvip) [inline plugins/home-page/package.json:24]: Good catch on the version 👍, I wonder if lerna missed bumping them because there were duplicates? 🤔

**Review by Rugvip — APPROVED** [review]
Awesome work, thanks! 👍 

I'm actually not sure whether types should always be dev deps or if it makes sense to have them as deps if we're reexporting them. Happy to merge this either way and figure that out separately.

**Comment by stefanalund** [comment]
@Rugvip should we merge?

**Comment by Rugvip** [comment]
Yep :p

**Pull request merged**